### ISSUE TITLE
Big bundle size

### ISSUE THREAD

**Issue by nicgirault** [i18next/i18next-browser-languageDetector#254]
## 💥 Regression Report

After upgrading from 6.0.1 to 6.1.3, the bundle size of i18next-browser-languageDetector is huge.

<img width="1148" alt="image" src="https://user-images.githubusercontent.com/551837/153578985-7dd975ed-2a2e-4d25-8ce6-0fe52be14fa1.png">

**Comment by adrai** [comment]
for example, [here](https://github.com/i18next/react-i18next/tree/master/example/react) it looks like this:
<img width="1501" alt="image" src="https://user-images.githubusercontent.com/1086194/153581203-ce521e7f-5231-4873-a8e2-2ced9471ae88.png">

**Comment by nicgirault** [comment]
I was wrong, the problem is already present in 6.0.1.
I use create-react-app v5.

My tsconfig.json look like this:

```
{
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "outDir": "build/dist",
    "module": "esnext",
    "target": "es5",
    "lib": ["dom", "dom.iterable", "esnext"],
    "sourceMap": true,
    "allowJs": true,
    "jsx": "react-jsx",
    "moduleResolution": "node",
    "rootDir": ".",
    "forceConsistentCasingInFileNames": true,
    "noImplicitReturns": true,
    "noImplicitThis": true,
    "noImplicitAny": true,
    "skipLibCheck": true,
    "strictNullChecks": true,
    "suppressImplicitAnyIndexErrors": true,
    "noUnusedLocals": false,
    "downlevelIteration": true,
    "esModuleInterop": true,
    "strict": true,
    "resolveJsonModule": true,
    "isolatedModules": true,
    "noEmit": true,
    "noFallthroughCasesInSwitch": true,
    "baseUrl": "src"
  },
  "exclude": [
    "node_modules"
  ],
  "include": ["src"]
}
```

 and I import like this:

```
import LanguageDetector from 'i18next-browser-languagedetector'
```

Does it answer your question?

**Comment by adrai** [comment]
Can you create a reproducible example based on https://github.com/i18next/react-i18next/tree/master/example/react ?

**Comment by adrai** [comment]
I tried to reproduce it, but even with your tsconfig.json file, I'm not able: https://github.com/adrai/react-i18next-ts

**Comment by chybisov** [comment]
@adrai hey, I have the same issue 
<img width="1708" alt="image" src="https://github.com/i18next/i18next-browser-languageDetector/assets/18644653/c088b7f6-4b7e-4e4e-b491-78436680ec0e">


You can find a reproducible example here https://github.com/lifinance/widget/tree/main/packages/widget-playground

**Comment by adrai** [comment]
sorry no idea 🤷‍♂️

**Comment by adrai** [comment]
No, that dependency is used.... check your installed `node_modules/i18next-browser-languagedetector/dist/esm/i18nextBrowserLanguageDetector.js` file
